packages/socket-mode/README.md

```diff
@@ -0,0 +1,245 @@
+# Slack Socket Mode
+
+[![build-ci](https://github.com/slackapi/node-slack-sdk/workflows/CI%20Build/badge.svg)](https://github.com/slackapi/node-slack-sdk/actions?query=workflow%3A%22CI+Build%22)
+<!-- TODO: per-flag badge https://docs.codecov.io/docs/flags#section-flag-badges-and-graphs -->
+[![codecov](https://codecov.io/gh/slackapi/node-slack-sdk/branch/master/graph/badge.svg)](https://codecov.io/gh/slackapi/node-slack-sdk)
+<!-- TODO: npm versions with scoped packages: https://github.com/rvagg/nodei.co/issues/24 -->
+
+## Installation
+
+```shell
+$ npm install @slack/socket-mode
+```
+
+<!-- START: Remove before copying into the docs directory -->
```

Comment:
??

Reply:
That comment exists in all of the packages here